### PR TITLE
[TOD] Projects folder for tod_simulator + scripts + documentation

### PR DESCRIPTION
**Pull request by moyapchen**
See the long `README.md` file.

I'd tested the two mega-scripts (`tod_distributed_uber{,_multiwoz}_script.py`) on the FAIR cluster a few weeks ago and haven't since, but I have no idea how well/not well those would run outside of the FAIR cluster anyhow, so... 

(Incidentally, there's lot of repeated-looking things in the `tod_distributed..` scripts; I'm actually fairly convinced abstracting most of it out would just be another level of indirection, so didn't bother. Can if it seems egregiously confusing though.)

### CODE REVIEW

File: projects/tod_simulator/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Task Oriented Dialogue (TOD) Script + World
+
```
Comment (stephenroller) [inline projects/tod_simulator/README.md:2]: start off with a link to the paper, the abstract, and a bibtex entry.

File: projects/tod_simulator/scripts/get_quick_eval_stats.py [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+            )
+            metrics = [
+                orig_path,
+                base,
+                str(ppl),
+                str(token_em),
+                str(jga),
+                str(jga_n),
+                str(jga_e),
+            ]
```
Comment (stephenroller) [inline projects/tod_simulator/scripts/get_quick_eval_stats.py:113]: eee fstring?

File: projects/tod_simulator/scripts/get_quick_eval_stats.py [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+                    if "1eb" in line:
+                        base = "AllNoApiRL1"
+                    if "076" in line:
+                        base = "AllYesApiRL1"
+                    if "79a" in line:
+                        base = "SysGsgdOnly"
+                    if "c31" in line:
+                        base = "SysGood"
+                    if "ba9" in line:
+                        base = "SysAll"
```
Comment (stephenroller) [inline projects/tod_simulator/scripts/get_quick_eval_stats.py:63]: I don't think this should stand

File: projects/tod_simulator/README.md (hunk 1 above)
Comment (moyapchen) [reply projects/tod_simulator/README.md:2]: Did in https://github.com/facebookresearch/ParlAI/pull/4201

File: projects/tod_simulator/scripts/get_quick_eval_stats.py (hunk 2 above)
Comment (moyapchen) [reply projects/tod_simulator/scripts/get_quick_eval_stats.py:113]: Yeah I can change it to that lol.

Did it this way initially cause having everything on separate lines was slightly more readable when I was initially going through this code...

File: projects/tod_simulator/scripts/get_quick_eval_stats.py (hunk 3 above)
Comment (moyapchen) [reply projects/tod_simulator/scripts/get_quick_eval_stats.py:63]: Good catch

**Review by stephenroller — APPROVED** [review]

**Pull request closed without merging**